apps/backend/src/modules/contributors/contributor.service.ts

```diff
@@ -164,6 +166,57 @@ class ContributorService implements Service {
 		return item.toObject();
 	}
 
+	public async split(
+		contributorId: number,
+		payload: ContributorSplitRequestDto,
+	): Promise<ContributorGetAllItemResponseDto | null> {
+		const currentContributorEntity =
+			await this.contributorRepository.find(contributorId);
+		const hasContributor = currentContributorEntity !== null;
+
+		if (!hasContributor) {
+			throw new ContributorError({
+				message: ExceptionMessage.CONTRIBUTOR_NOT_FOUND,
+				status: HTTPCode.NOT_FOUND,
+			});
+		}
+
+		const currentContributor = currentContributorEntity.toObject();
+
+		if (currentContributor.gitEmails.length <= SINGLE_ITEM) {
```

Comment:
let's create a more specific constant for this
```suggestion
		if (currentContributor.gitEmails.length <= MIN_GIT_EMAILS_LENGTH_FOR_SPLIT) {
```